test/jdk/jdk/crac/java/net/InetAddress/ResolveTest.java

```diff
@@ -96,11 +98,16 @@ private static Future<?> startTestProcess() throws Exception {
         cmd.addAll(Arrays.asList("--name", CONTAINER_NAME));
         cmd.add(imageName);
         // Checkpointing does not work for PID 1, therefore we add an intermediary bash process
+        // At the same time docker --init is not sufficient; since `tini` does not wait for *all*
+        // children to finish, just the main process, creating /cr/cppath might race and we would
+        // not be able to restore the process.
+        // This workaround should not be necessary when https://github.com/openjdk/crac/pull/46
+        // is integrated.
```

Comment:
I propose to wait until #46 is integrated and remove the mention (and workaround)

Reply:
I would integrate code when it's ready reagardless of other PRs (haven't actually checked if this works with #46). This code will be anyway rewritten with #50